File: app/src/main/java/com/likhit/polis/ui/home/HomeFragementListener.java
package com.likhit.polis.ui.home;

import com.likhit.polis.data.models.Policy;

public interface HomeFragementListener {

    void launchHome();

    void launchUserDetails();

    void launchQA();

    void launchRecommendations();

    void launchPolicyDetails(Policy policy);

}

File: app/src/main/java/com/likhit/polis/ui/home/HomeFragementListener.kt
package com.likhit.polis.ui.home

import com.likhit.polis.data.models.Policy

interface HomeFragementListener {

    fun launchHome()

    fun launchUserDetails()

    fun launchQA()

    fun launchRecommendations(isPolicies: Boolean)

    fun launchPolicyDetails(policy: Policy)

}
